lib/sord/type_converter.rb

```diff
@@ -195,9 +204,20 @@ def self.yard_to_sorbet(yard, item = nil, replace_errors_with_untyped = false, r
         return from_yaml.class.to_s \
           if [Symbol, Float, Integer].include?(from_yaml.class)
 
-        Logging.warn("#{yard.inspect} does not appear to be a type", item)
-        replace_errors_with_untyped ? "T.untyped" : "SORD_ERROR_#{yard.gsub(/[^0-9A-Za-z_]/i, '')}"
+        return handle_sord_error(yard.to_s, "#{yard.inspect} does not appear to be a type", item, replace_errors_with_untyped)
       end
     end
+
+    # Handles SORD_ERRORs.
+    #
+    # @param [String] name
+    # @param [String] log_warning
+    # @param [YARD::CodeObjects::Base] item
+    # @param [Boolean] replace_errors_with_untyped
+    # @return [String]
+    def self.handle_sord_error(name, log_warning, item, replace_errors_with_untyped)
```

Comment:
I don't love these parameter names, if you have any better ideas I'm open to them.